config/database.yml

```diff
@@ -11,14 +11,12 @@ default: &default
 
 development:
   <<: *default
-  database: education_development
```

Comment:
No, because I didn't notice they were using the same db because I've got no seed data. I added the db name to the config.